lib/twingly/url_cache.rb

```diff
@@ -5,25 +5,33 @@
 
 module Twingly
   class UrlCache
+    class Error < StandardError; end
+    class ServerError < Error; end
```

Comment:
perhaps too much ceremony to move these to an `errors.rb` file? 😄

Reply:
The gem only has one file (+ a `version.rb` file) and I didn't think it was worth introducing another one for just two lines :)

Reply:
Sure :)